docs/section4/4-1.md

```diff
@@ -33,7 +33,7 @@
             - Check if schedules match each hour of the year, if there is a mismatch then set the schedule_mismatch boolean variable equal to TRUE: `if thermostat_heating_stpt_sch_b.hourly_values(x) != thermostat_heating_stpt_sch_p.hourly_values(x):schedule_mismatch = TRUE`
     **Rule Assertion:**
     - Case 1: For each zone if the baseline and proposed heating and cooling thermostat schedule match: `if schedule_mismatch == FALSE: PASS`
```

Comment:
Should be  ````if schedule_mismatch == **TRUE**: PASS````? because the text says ```For each zone if the baseline and proposed heating and cooling thermostat schedule match```

Reply:
@yunjoonjung-PNNL the boolean name is schedule_**mis**match so what is there is correct.

Reply:
I missed this part. Thank you for your confirmation.

Reply:
No problem!